synapse/axon.py

```diff
@@ -18,6 +18,8 @@
 import synapse.lib.thishost as s_thishost
 import synapse.lib.thisplat as s_thisplat
 
+from synapse.models.axon import AxonMod as axon_model
```

Comment:
This mixed capitalization is weird.  Why not just do
```
import synaspe.models.axon as s_models_axon
```
 then you can refer to s_models_axon.AxonMod directly?